src/main/java/com/google/devtools/build/lib/skyframe/IgnoredPackagePrefixesFunction.java

```diff
@@ -82,7 +86,19 @@ public SkyValue compute(SkyKey key, Environment env)
       }
 
       if (repositoryName.isMain()) {
+        PathFragment vendorDir = null;
+        if (VENDOR_DIRECTORY.get(env).isPresent()) {
+          vendorDir = VENDOR_DIRECTORY.get(env).get().asFragment();
+          // Vendor dir should have been resolved to an absolute path.
+          Preconditions.checkArgument(vendorDir.isAbsolute());
```

Comment:
this check is unnecessary as `VENDOR_DIRECTORY` has type `Optional<Path>`, and `Path` is always absolute (you literally just called `asFragment()` above :)

Reply:
Thanks!